text/0000-distribution-team.md

```diff
@@ -0,0 +1,70 @@
+# Meta
+[meta]: #meta
+- Name: Create a Distribution Team
+- Start Date: 2020-09-03
+- Author(s): [@jkutner](@jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal to create a Distribution sub-team in the project governance.
+
+# Definitions
+[definitions]: #definitions
+
+- *sub-team* - Sub-teams are responsible for narrower sets of concerns related to specific aspects of the project. Each sub-team will include at least one core team member to help align with the broader roadmap.
+- *distribution* - relates to the delivery, discovery, and integration of buildpacks.
+
+# Motivation
+[motivation]: #motivation
+
+We have two repositories that are not owned by a specific sub-team.
+
+# What it is
+[what-it-is]: #what-it-is
+
```

Comment:
Can you also add the description/charter from the [community PR](https://github.com/buildpacks/community/pull/34/files):

> The Distribution team is responsible for maintaining tools and services that support the distribution, discovery, and integration of buildpacks.